litr-muxers/src/main/cpp/CMakeLists.txt

```diff
@@ -48,6 +48,10 @@ else()
 
     set(ffmpeg_location "${ffmpeg_bundled_source}")
     set(ffmpeg_binaries "${CMAKE_CURRENT_SOURCE_DIR}/ffmpeg_bundled/android-libs/${ANDROID_ABI}")
+
+    # Include additional headers that would be generated via the configure (which won't be there in
+    # the downloaded artifact.
+    include_directories("${CMAKE_CURRENT_SOURCE_DIR}/ffmpeg_bundled/extra")
```

Comment:
Hopefully the last cmake change 😅 Turned out that the `configure` auto-generated this header, so included in the repo to augment the vanilla source that we download (when the user doesn't have their own checkout).